app/src/debug/java/pl/gov/mc/protego/ui/base/BaseActivity.kt

```diff
@@ -0,0 +1,22 @@
+package pl.gov.mc.protego.ui.base
+
+import android.os.Bundle
+import kotlinx.android.synthetic.main.toolbar.*
+
+abstract class BaseActivity : BaseVariantActivity() {
```

Comment:
To chyba powinno być w main?

Reply:
Tak, to prawda, budowałem wszystkie warianty i działało, w sumie nie wiem dlaczego 🤔 
Przenoszę

Reply:
Ok już wiem dlaczego, AndroidStudio podczas tworzenia pliku `BaseActivity` utworzyło mi go w każdym variant directory  osobno zamiast tylko w main 🤔